sdk/identity/azure-identity/src/main/java/com/azure/identity/AzureCliCredential.java

```diff
@@ -32,6 +32,8 @@
 
     @Override
     public Mono<AccessToken> getToken(TokenRequestContext request) {
-        return identityClient.authenticateWithAzureCli(request);
+        return identityClient.authenticateWithAzureCli(request)
+                .onErrorMap(t -> new CredentialUnavailableException(
```

Comment:
This onErrorMap should be removed.
We don't want to map any exception to CredentialUnavailableException.